plugins/search-backend-module-elasticsearch/api-report.md

```diff
@@ -15,17 +15,44 @@ import { Logger } from 'winston';
 import { SearchEngine } from '@backstage/plugin-search-common';
 import { SearchQuery } from '@backstage/plugin-search-common';
 
-// Warning: (ae-missing-release-tag) "ElasticSearchClientOptions" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-// Warning: (ae-unresolved-link) The @link reference could not be resolved: The package "@backstage/plugin-search-backend-module-elasticsearch" does not have an export "ElasticSearchEngine"
-//
+// @public
+export type ConcreteElasticSearchQuery = {
```

Comment:
Feeling that everything else is so nicely prefixed, perhaps make this `ElasticSearchConcreteQuery`?

Reply:
Oh thats a great catch, I'll update that one! thanks @Rugvip

Reply:
LGTM, @emmaindal!  :shipit: